INSTALL.md

```diff
@@ -0,0 +1,291 @@
+#Installation Instructions
+
+
+### Table of Contents
+
+**[Overview of Directories](#overview-of-directories)**
+
+**[Unpacking](#unpacking)**
+
+**[Setup](#setup)**
+
+**[Setting up Access Control](#setting-up-access-control)**
+
+**[Upgrading](#upgrading)**
+
+**[FAQ](#faq)**
+
+
+##  Overview of Directories
+
+NOTE: Most recent documentation can be found on the online documentation at [LibreHealth](http://librehealth.io/).
+
+accounting: Contains information and scripts to support SQL-Ledger
+contrib: Contains many user-contributed encounter forms and utilities
```

Comment:
Sorry did not catch this on the last pass. We do not have accounting any more that code is obsolete and was removed